app/models/user_access.rb

```diff
@@ -83,6 +83,18 @@ def accessible_users(action)
       .where(phone_number_authentications: {registration_facility_id: accessible_facilities(action)})
   end
 
+  def accessible_protocols(_action)
```

Comment:
Hmm, now that we have other models that utilize `accessible_*` methods, should we rename the `resources_for` method to `regions_for`?

Reply:
Not fully convinced we should do this, would mean we should also change `access.resource` to `access.region`. Lets leave it for later? Since `resources_for` is a private method, we can change this easily.

Reply:
Ah I see. Yup sounds good to me 👍